src/main/java/com/atharvadholakia/password_manager/controller/CredentialController.java

```diff
@@ -20,6 +20,7 @@ public CredentialController(CredentialService credentialService) {
 
   @PostMapping
   public ResponseEntity<Credential> createCredential(@Valid @RequestBody Credential credential) {
+    credential.setServicename(credential.getServiceName().trim());
```

Comment:
how about having this `trim` while instantiating `credential` in credential.java line 46? It will be make serviceName consistent irrespective of an object being created from anywhere in the codebase.